openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -119,6 +119,15 @@ def setUpClass(cls):
         cls.heketi_cli_user = heketi_config['heketi_cli_user']
         cls.heketi_cli_key = heketi_config['heketi_cli_key']
 
+        # Initializes registry heketi config variables
+        heketi_config = cls.openshift_config['registry_heketi_config']
```

Comment:
missing `registry` prefix in variable name ?

Reply:
Done